happi/device.py

```diff
@@ -394,7 +394,7 @@ class Device(HappiItem):
     args = copy.copy(HappiItem.args)
     args.default = ['{{prefix}}']
     kwargs = copy.copy(HappiItem.kwargs)
-    args.default = {'name': '{{name}}'}
+    kwargs.default = {'name': '{{name}}'}
```

Comment:
FYI happi has been adding devices with the wrong default args and kwargs for at least a year